src/DynamoCore/Models/DynamoModelCommands.cs

```diff
@@ -271,6 +271,10 @@ void MakeConnectionImpl(MakeConnectionCommand command)
                     EndShiftReconnections(nodeId, command.PortIndex, command.Type);
                     break;
 
+                case MakeConnectionCommand.Mode.BeginCtrlConnection:
+                    BeginCtrlConnection(nodeId, command.PortIndex, command.Type);
+                    break;
+
                 case MakeConnectionCommand.Mode.EndAndStartCtrlConnection:
                     EndAndStartCtrlConnection(nodeId, command.PortIndex, command.Type);
                     break;
```

Comment:
Hi @yeexinc, after further considerations I think better names for the modes would be `BeginDuplicateConnection` and `BeginCreateConnections`. Please also rename the methods to reflect this.